config/routes.rb

```diff
@@ -186,6 +186,10 @@
     get '/sign_up' => 'clearance/users#new', as: 'sign_up' if Clearance.configuration.allow_sign_up?
   end
 
+  ################################################################################
+  # high_voltage static routes
+  get 'pages/*id' => 'high_voltage/pages#show', constraints: { id: /(about|data|download|faq|migrate|security|sponsors)/ }, as: :page
```

Comment:
Is there some way to build the list of pages at boot time instead of hardcoding them?

Reply:
`HighVoltage.page_ids` :question:

Reply:
Updated to use `page_ids`. Technically, `constraints` support array as value as well[[1](https://github.com/rails/rails/pull/32025/files#diff-e304f21f210ada0bd90fb1990912b08eR4503)], however, I get `ActionView::Template::Error: No route matches` when I use that. 
Also, I tried to reproduce this with [high-voltage spec](https://travis-ci.org/github/sonalkr132/high_voltage/jobs/681659014#L363) but there it wasn't raising the error we are getting.